lib/iron_bank/authentications/cookie.rb

```diff
@@ -54,9 +54,9 @@ def authenticate
 
       def connection
```

Comment:
Do we know why we create our own connection here, instead of leveraging `IronBank::Client`? cc: @mickaelpham, @mustafaturan

Reply:
Late to answer, but this connection is used for the initial authentication cookie or token. The payload for this connection is different (`url_encoded` instead of `json`).